src/inputs/Table/Table.js

```diff
@@ -168,21 +173,46 @@ export const Table = (props) => {
           buildErrorsCountLabel={buildErrorsPanelTitle}
         />
       )}
-      <div data-cy="grid" id="grid-container" style={dimensions} className={agTheme}>
```

Comment:
Was there a reason to remove the id `id="grid-container"`?
Removing this id could break some tests written by users, so unless the id has become obsolete, we should keep it :slightly_smiling_face: